doc/wakaama_features.rst

```diff
@@ -0,0 +1,213 @@
++                            +----------------------------------------+------------------+-------------------------------+
+|                            | Firmware Update Object (ID:5)          | as example       |                               |
++                            +----------------------------------------+------------------+-------------------------------+
+|                            | Location Object (ID:6)                 | as example       |                               |
++                            +----------------------------------------+------------------+-------------------------------+
+|                            | Connectivity Statistics Object (ID:7)  | v_1.0            |                               |
++                            +----------------------------------------+------------------+-------------------------------+
+|                            | OSCORE Object (ID:21)                  | none             |                               |
++                            +----------------------------------------+------------------+-------------------------------+
+|                            | LwM2M COSE (ID:23)                     | none             |                               |
++                            +----------------------------------------+------------------+-------------------------------+
+|                            | MQTT Server (ID:24)                    | none             |                               |
++                            +----------------------------------------+------------------+-------------------------------+
+|                            | LwM2M Gateway (ID:25)                  | none             |                               |
++                            +----------------------------------------+------------------+-------------------------------+
+|                            | LwM2M Gateway Routing (ID:26)          | none             |                               |
++                            +----------------------------------------+------------------+-------------------------------+
+|                            | 5G-NR Connectivity Object (ID:27)      | none             |                               |
++----------------------------+----------------------------------------+------------------+-------------------------------+
+| Interaction with Server    | Multiple LwM2M Servers                 | v_1.0            |                               |
++                            +----------------------------------------+------------------+-------------------------------+
+|                            | LwM2M bootstrap-server                 | v_1.0            |                               |
++----------------------------+----------------------------------------+------------------+-------------------------------+
+
+
+Overview of LwM2M feature as specified by the OMA:
```

Comment:
Very nice to have, but... is this not already existing and maintained by someone else out there?

Reply:
I could not find any overview like this.